fbw-a32nx/src/systems/instruments/src/SD/Pages/Crz/Crz.tsx

```diff
@@ -119,38 +119,35 @@ export const OilComponent = () => {
 };
 
 export const PressureComponent = () => {
-    const [landingElevDialPosition] = useSimVar('L:XMLVAR_KNOB_OVHD_CABINPRESS_LDGELEV', 'Number', 100);
-    // FIXME Use CPC landing elev ARINC vars when made and get them via SDACs when made
-    const [landingRunwayElevation] = useSimVar('L:A32NX_PRESS_AUTO_LANDING_ELEVATION', 'feet', 1000);
+    const cpc1DiscreteWord = useArinc429Var('L:A32NX_PRESS_CPC_1_DISCRETE_WORD');
+    const cpc2DiscreteWord = useArinc429Var('L:A32NX_PRESS_CPC_2_DISCRETE_WORD');
+
+    const cpcDiscreteWordToUse = cpc1DiscreteWord.getBitValueOr(11, false) ? cpc1DiscreteWord : cpc2DiscreteWord;
+
+    const landingElevationIsMan = cpcDiscreteWordToUse.getBitValueOr(17, false);
+    const landingElevation = useArinc429Var('L:A32NX_PRESS_LANDING_ELEVATION', 1000);
+
     const [autoMode] = useSimVar('L:A32NX_OVHD_PRESS_MODE_SEL_PB_IS_AUTO', 'Bool', 1000);
     const [ldgElevMode, setLdgElevMode] = useState('AUTO');
     const [ldgElevValue, setLdgElevValue] = useState('XX');
     const [cssLdgElevName, setCssLdgElevName] = useState('green');
-    const [landingElev] = useSimVar('L:A32NX_OVHD_PRESS_LDG_ELEV_KNOB', 'feet', 100);
-    const [cabinAlt] = useSimVar('L:A32NX_PRESS_CABIN_ALTITUDE', 'feet', 500);
-    const [cabinVs] = useSimVar('L:A32NX_PRESS_CABIN_VS', 'feet per minute', 500);
-    const [deltaPsi] = useSimVar('L:A32NX_PRESS_CABIN_DELTA_PRESSURE', 'psi', 1000);
+    const cabinAlt = useArinc429Var('L:A32NX_PRESS_CABIN_ALTITUDE', 500);
+    const cabinVs = useArinc429Var('L:A32NX_PRESS_CABIN_VS', 500);
+    const deltaPsi = useArinc429Var('L:A32NX_PRESS_CABIN_DELTA_PRESSURE', 1000);
 
     const vsx = 440;
     const y = 385;
     const radius = 50;
 
-    const deltaPress = splitDecimals(deltaPsi);
+    const deltaPress = splitDecimals(deltaPsi.value);
```

Comment:
**question:** it seems like in this page, there is no handling of the Arinc429Word actually holding a valid value. Are we missing some logic to check the SSM?

Reply:
This is a great point and something I thought about. The issue is that when both CPCs fail in the real aircraft, the data for the ECAM pages is taken from a backup pressure sensor which is not modelled at the moment. 

For now I've decided to take the values from the active CPC regardless of whether it is sending a Failure Warning signal (as forcing a failure doesn't affect the calculation of the values anyway).

Reply:
So, if I understand correctly, right now those vars would never have an SSM that isn't NormalOperation?

If that is right, let's add a TODO comment explaining this in the relevant UI files.

Reply:
After conversations with @lukecologne on discord I've changed this and added the analog signals coming from the manual partition of CPC1. I've had to use a couple of workarounds since we don't have the SDAC logic implemented. The manual partition at the moment sends cabin altitude and differential pressure, which is not true to real life (these should be calculated within the SDAC).